### ISSUE TITLE
Support ZMQ messaging 

### ISSUE THREAD

**Issue by ProPheT777** [Tolerance/Tolerance#51]
There is a plan to support ZMQ messaging?

**Comment by sroze** [comment]
Not at the moment, but it would be nice. If you are happy to help with that I can guide you through the related codebase in order to contribute.

**Comment by ProPheT777** [comment]
Yes, indeed I can have the use case and I'll help if we need it :+1:
